Updated user.js & antiadblock.js.

miccayo opened the pull request:
User.js is now more optimized (I assume) and now makes use of the /API/current_user endpoint.

Anyways, the code is now cleaner and more efficient, and the antiadblock code is also a bit more clean.

Tested in Chrome, and confirmed works with these changes.

Commit:
Update antiadblock.js

Commit:
Update antiadblock.js

Commit:
Update antiadblock.js

Commit:
Update antiadblock.js

Commit:
Rewrote user.js.

Speedier functionality and direct request to an API that supplies all of the required fields.

Tested and successful.

Commit:
Heck. Updated User.js

More efficient connection to /current_user.

Required fields are updated faster and require less jQuery to find/set.

Commit:
Merge branch 'master' of https://github.com/Kebabalicious/golden-brick

Peridax commented:
In the user.js rewrite you forgot to include the interval to keep updating the information.

Commit:
Update user.js

miccayo commented:
Apologies. Should be fixed now.

miccayo reviewed (commented):
Updated, added intervals.

Commit:
Clean up and minor fixes

Set the interval to clear outside and before the HTTP request, and placed the ban script inside the if statement when it checks if the user is logged in.
